doc/use_apps/verify_tx.rst

```diff
@@ -68,6 +68,20 @@ This means that the request may return ``202 Accepted`` at first, with a suggest
      'root': '06fef62c80b6471c7005c1b114166fd1b0e077845f5ad544ad4eea4fb1d31f78',
      'signature': 'MGQCMACklXqd0ge+gBS8WzewrwtwzRzSKy+bfrLZVx0YHmQvtsqs7dExYESsqrUrB8ZcKwIwS3NPKaGq0w2QlPlCqUC3vQoQvhcZgPHPu2GkFYa7JEOdSKLknNPHaCRv80zx2RGF'}
 
+Note that receipts over signature transactions are a special case, for example:
+
+.. code-block:: bash
+
+    $ curl -X GET "https://<ccf-node-address>/app/receipt?transaction_id=2.35" --cacert networkcert.pem --key user0_privk.pem --cert user0_cert.pem
+    
+    {'leaf': 'fdc977c49d3a8bdf986176984e9432a09b5f6fe0c04e0b1c2dd177c03fdca9ec',
+     'node_id': '06fef62c80b6471c7005c1b114166fd1b0e077845f5ad544ad4eea4fb1d31f78',
+     'proof': [],
+     'root': '06fef62c80b6471c7005c1b114166fd1b0e077845f5ad544ad4eea4fb1d31f78',
+     'signature': 'MGQCMACklXqd0ge+gBS8WzewrwtwzRzSKy+bfrLZVx0YHmQvtsqs7dExYESsqrUrB8ZcKwIwS3NPKaGq0w2QlPlCqUC3vQoQvhcZgPHPu2GkFYa7JEOdSKLknNPHaCRv80zx2RGF'}
+
+The proof is empty, and the 'leaf' and 'root' fields are both set to the value being signed, which is the root of the Merkle Tree covering all transactions until the signature.
```

Comment:
We could be a little more explicit about the implications of this. Namely, you can verify both with the same code, starting from the leaf and iterating through the proof to produce the root, but this isn't really a leaf and you shouldn't use it as such if you're trying to reconstruct a full merkle tree.

Reply:
Ok, I've extended the description accordingly.